tests/e2e/specs/out-of-game/home.spec.js

```diff
@@ -12,12 +12,22 @@ function setup() {
   cy.signupPlayer(playerUsername, playerPassword);
   cy.vueRoute('/');
 }
+
 function assertSuccessfulJoin(gameState) {
   expect(gameState.id).to.not.eq(null);
   cy.url().should('include', '/lobby/');
   cy.contains('h1', `Lobby for ${gameState.name}`);
 }
 
+function assertParentInputToggle(selector, checked = false) {
```

Comment:
Nice; this DRY's things up a bit. Why `Parent`? Maybe just `toggleInput()`?

Reply:
Because it doesn't just toggle an input. It's specifically a wrapper around toggling a parent (like vuetify provides) when we need to check the inner input. Check below, we add `${selector} input` to the selector string. Additionally we aren't just toggling it, we're asserting the state AND toggling it.

Reply:
But we don't have any inputs that aren't wrapped, so from our consumption's standpoint the component sortof just is a sparkling input. Like it doesn't make sense for the 'switch' or checkbox or w/e to be checked if it's inner input isn't b/c the component's state is effectively computed from the input itself, right?

As for asserting, I could go either way, but it seems reasonable to me to say that the helper function serves the purpose of toggling an input and because this is a testing context we assert it's state before and after every time we do that